src/components/LeftSidebar/NewGroupConversation/SetContacts/SetContacts.vue

```diff
@@ -121,11 +143,17 @@ export default {
 <style lang="scss" scoped>
 .set-contacts {
 	position: relative;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	overflow: hidden;
 	&__input {
 		width: 100%;
 		font-size: 16px;
 		padding-left: 28px;
```

Comment:
This feels weird/out of place

Reply:
Why does it feel weird?? It's needed, try and toggle it off in the devtools!

Reply:
For the name it looks good:

![Bildschirmfoto von 2020-03-24 07-42-45](https://user-images.githubusercontent.com/213943/77396097-561c1800-6da3-11ea-8414-ede3124f6981.png)

But not for searching participants:

![Bildschirmfoto von 2020-03-24 07-43-28](https://user-images.githubusercontent.com/213943/77396118-60d6ad00-6da3-11ea-8c4a-bfe7f788b8d8.png)

Reply:
You're missing the search icon, is that what you're referring to?

Reply:
Uhh  I gues sso, but why would this one field have a search icon and the other search7filter fields not? (Can we skip it for now and in case we want the icon do it in a new PR for all search fields?)

Reply:
Nope, this has been done as per Jan's specs and it's already released in talk 8. We need to bring it in the other search fields as well... Maybe do a single input component?